### ISSUE TITLE
Wake Screen with New Message

### ISSUE THREAD

**Issue by HonestAbe7** [FossifyOrg/Messages#63]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Messages/issues) or [closed](https://github.com/FossifyOrg/Messages/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Messages/discussions) or [closed](https://github.com/FossifyOrg/Messages/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Messages/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

I would like my phone to wake when I get a new text message.

### Why do you want this feature?

This would allow me to know if I got a new message without looking in my notifications.

**Comment by inson1** [comment]
how wake up? open the message in the app?

**Comment by HonestAbe7** [comment]
My phone wakes us, the screen turns on when I get a new message if I use the QKSMS message.

**Comment by mat999io** [comment]
They ignore this issue without giving any response.

**Comment by Aga-C** [comment]
It's not ignored - it's open and `needs triage` is taken off. Fossify is a huge project, and we don't have a possibility to address every possible issue in every app in a short time.

**Comment by naveensingh** [comment]
It turns out, there's an OS level option for this on devices that support it:

- **Settings** ➜ **Display** ➜ **Lock screen** ➜ **Wake screen for notifications**

(Sometimes, located under notification settings or buried under **More settings**. HyperOS calls it **Notification effects**)

An in-app toggle won't be added because it would be duplicating a system-level option and [hard on the battery](https://developer.android.com/topic/performance/vitals/excessive-wakelock).

**Comment by antodc** [comment]
> An in-app toggle won't be added because it would be duplicating a system-level option and [hard on the battery](https://developer.android.com/topic/performance/vitals/excessive-wakelock).

Would this increase in battery consumption happen also if this feature is made optional and the option is turned off? I am asking because in my case the OS level option you mentioned does not seem to be available and sometimes (eg, when waiting for important messages) this feature could be useful also to me.

**Comment by naveensingh** [comment]
> Would this increase in battery consumption happen also if this feature is made optional and the option is turned off?

No, only when turned on and subject to the device-specific hardware optimizations. The use of wake locks (required for this feature) is unlikely to cause a Google Play policy violation, but this is still a system-level option, and the goal here is to avoid adding too many bells and whistles. 

Getting it to work reliably is [another issue](https://dontkillmyapp.com/).

> the OS level option you mentioned does not seem to be available

The option is usually made available, but it's up to the OEM. They may decide that such an option would affect battery life negatively, depending on the type of display, etc., but I'm speculating here.
